runtime/dart_init.cc

```diff
@@ -78,18 +78,18 @@ const char kSnapshotAssetKey[] = "snapshot_blob.bin";
 
 namespace {
 
-static const char* kDartMirrorsArgs[] = {
+// Arguments passed to the Dart VM in all configurations.
+static const char* kDartLanguageArgs[] = {
     "--enable_mirrors=false",
+    "--background_compilation",
+    "--await_is_keyword",
+    "--warn_super",
```

Comment:
It's a warning for when you have `super` not last in your initializer list. Pretty sure we always want it last; IIRC strong mode is going to require that anyway.